### ISSUE TITLE
About paper《Tailoring Instructions to Student’s Learning Levels Boosts Knowledge Distillation》 github demo

### ISSUE THREAD

**Issue by oooo111** [twinkle0331/LGTM#2]
I use setting --output ./outputs  and I also create outputs folders in my dictionary。But have error is --t_learning_rate: command not found. How to solve this problem？

**Comment by Harry-zzh** [comment]
Hi, could you give the complete command you used?

**Comment by oooo111** [comment]
python run_glue.py \
    --model_name_or_path google/bert_uncased_L-6_H-768_A-12 \
    --teacher_model bert-base-uncased \
    --task_name mnli \
    --per_device_train_batch_size 32 \
    --per_device_eval_batch_size 32 \
    --learning_rate 3e-05 \ # {3e-05,5e-05,1e-04}
    --t_learning_rate 3e-05 \
    --alpha_kd 1.0 \
    --temperature 1.0 \
    --num_train_epochs 6 \
    --output_dir ./outputs \
    --eval_steps 1000 \
    --do_train \
    --do_eval \
    --train_teacher \
    --init_classifier_to_zero \
    --use_lgtm

I also create outputs makedir in my dictionary.

**Comment by Harry-zzh** [comment]
I think you could try the following command:

```sh
  python run_glue.py \
      --model_name_or_path google/bert_uncased_L-6_H-768_A-12 \
      --teacher_model bert-base-uncased \
      --task_name mnli \
      --per_device_train_batch_size 32 \
      --per_device_eval_batch_size 32 \
      --learning_rate 3e-05 \
      --t_learning_rate 3e-05 \
      --alpha_kd 1.0 \
      --temperature 1.0 \
      --num_train_epochs 6 \
      --output_dir ./outputs \
      --eval_steps 1000 \
      --do_train \
      --do_eval \
      --train_teacher \
      --init_classifier_to_zero \
      --use_lgtm
   ```

The "#{3e-05,5e-05,1e-04}" is just a note indicating that the choice of the learning rate. If you copy the command to the terminal without deleting this note , it will raises an error.

And I will update the run_glue.py that one could avoid creating the output dir by hand. Thank you for your advice.
